src/frontend/components/DashboardVideoLiveStopButton/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { Button } from 'grommet';
+import React, { useState } from 'react';
+import { defineMessages, FormattedMessage } from 'react-intl';
+import { Redirect } from 'react-router-dom';
+import styled from 'styled-components';
+
+import { stopLive } from '../../data/sideEffects/stopLive';
+import { useVideo } from '../../data/stores/useVideo';
+import { Video } from '../../types/tracks';
+import { Nullable } from '../../utils/types';
+import { dashboardButtonStyles } from '../DashboardPaneButtons';
+import { ERROR_COMPONENT_ROUTE } from '../ErrorComponent/route';
+import { Loader } from '../Loader';
+
+const messages = defineMessages({
+  stopLive: {
+    defaultMessage: 'stop streaming',
+    description: 'Stop a video streaming.',
+    id: 'components.DashboardVideoLiveStopButton.startLive',
+  },
+});
+
+const StartLiveButton = styled(Button)`
```

Comment:
Instead of share styles, you could create a generic DashboardButton component and wrap it in withLink HOC when needed or customize it.

I think it's better to create a reusable component than share css-in-js style between components.